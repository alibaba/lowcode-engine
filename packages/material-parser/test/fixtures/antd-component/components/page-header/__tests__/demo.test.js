import demoTest from '../../../tests/shared/demoTest';

demoTest('page-header');
