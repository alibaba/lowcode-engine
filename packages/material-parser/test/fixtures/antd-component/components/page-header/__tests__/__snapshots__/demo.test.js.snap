// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ./components/page-header/demo/actions.md correctly 1`] = `
<div>
  <div
    class="ant-page-header site-page-header ant-page-header-ghost"
  >
    <div
      class="ant-page-header-heading"
    >
      <div
        class="ant-page-header-heading-left"
      >
        <div
          class="ant-page-header-back"
        >
          <div
            aria-label="Back"
            class="ant-page-header-back-button"
            role="button"
            style="border:0;background:transparent;padding:0;line-height:inherit;display:inline-block"
            tabindex="0"
          >
            <span
              aria-label="arrow-left"
              class="anticon anticon-arrow-left"
              role="img"
            >
              <svg
                aria-hidden="true"
                class=""
                data-icon="arrow-left"
                fill="currentColor"
                focusable="false"
                height="1em"
                viewBox="64 64 896 896"
                width="1em"
              >
                <path
                  d="M872 474H286.9l350.2-304c5.6-4.9 2.2-14-5.2-14h-88.5c-3.9 0-7.6 1.4-10.5 3.9L155 487.8a31.96 31.96 0 000 48.3L535.1 866c1.5 1.3 3.3 2 5.2 2h91.5c7.4 0 10.8-9.2 5.2-14L286.9 550H872c4.4 0 8-3.6 8-8v-60c0-4.4-3.6-8-8-8z"
                />
              </svg>
            </span>
          </div>
        </div>
        <span
          class="ant-page-header-heading-title"
          title="Title"
        >
          Title
        </span>
        <span
          class="ant-page-header-heading-sub-title"
          title="This is a subtitle"
        >
          This is a subtitle
        </span>
      </div>
      <span
        class="ant-page-header-heading-extra"
      >
        <button
          class="ant-btn"
          type="button"
        >
          <span>
            Operation
          </span>
        </button>
        <button
          class="ant-btn"
          type="button"
        >
          <span>
            Operation
          </span>
        </button>
        <button
          class="ant-btn ant-btn-primary"
          type="button"
        >
          <span>
            Primary
          </span>
        </button>
      </span>
    </div>
    <div
      class="ant-page-header-content"
    >
      <div
        class="ant-descriptions ant-descriptions-small"
      >
        <div
          class="ant-descriptions-view"
        >
          <table>
            <tbody>
              <tr
                class="ant-descriptions-row"
              >
                <td
                  class="ant-descriptions-item"
                  colspan="1"
                >
                  <span
                    class="ant-descriptions-item-label ant-descriptions-item-colon"
                  >
                    Created
                  </span>
                  <span
                    class="ant-descriptions-item-content"
                  >
                    Lili Qu
                  </span>
                </td>
                <td
                  class="ant-descriptions-item"
                  colspan="1"
                >
                  <span
                    class="ant-descriptions-item-label ant-descriptions-item-colon"
                  >
                    Association
                  </span>
                  <span
                    class="ant-descriptions-item-content"
                  >
                    <a>
                      421421
                    </a>
                  </span>
                </td>
                <td
                  class="ant-descriptions-item"
                  colspan="1"
                >
                  <span
                    class="ant-descriptions-item-label ant-descriptions-item-colon"
                  >
                    Creation Time
                  </span>
                  <span
                    class="ant-descriptions-item-content"
                  >
                    2017-01-10
                  </span>
                </td>
              </tr>
              <tr
                class="ant-descriptions-row"
              >
                <td
                  class="ant-descriptions-item"
                  colspan="1"
                >
                  <span
                    class="ant-descriptions-item-label ant-descriptions-item-colon"
                  >
                    Effective Time
                  </span>
                  <span
                    class="ant-descriptions-item-content"
                  >
                    2017-10-10
                  </span>
                </td>
                <td
                  class="ant-descriptions-item"
                  colspan="2"
                >
                  <span
                    class="ant-descriptions-item-label ant-descriptions-item-colon"
                  >
                    Remarks
                  </span>
                  <span
                    class="ant-descriptions-item-content"
                  >
                    Gonghu Road, Xihu District, Hangzhou, Zhejiang, China
                  </span>
                </td>
              </tr>
            </tbody>
          </table>
        </div>
      </div>
    </div>
  </div>
  <br />
  <div
    class="ant-page-header ant-page-header-ghost"
  >
    <div
      class="ant-page-header-heading"
    >
      <div
        class="ant-page-header-heading-left"
      >
        <div
          class="ant-page-header-back"
        >
          <div
            aria-label="Back"
            class="ant-page-header-back-button"
            role="button"
            style="border:0;background:transparent;padding:0;line-height:inherit;display:inline-block"
            tabindex="0"
          >
            <span
              aria-label="arrow-left"
              class="anticon anticon-arrow-left"
              role="img"
            >
              <svg
                aria-hidden="true"
                class=""
                data-icon="arrow-left"
                fill="currentColor"
                focusable="false"
                height="1em"
                viewBox="64 64 896 896"
                width="1em"
              >
                <path
                  d="M872 474H286.9l350.2-304c5.6-4.9 2.2-14-5.2-14h-88.5c-3.9 0-7.6 1.4-10.5 3.9L155 487.8a31.96 31.96 0 000 48.3L535.1 866c1.5 1.3 3.3 2 5.2 2h91.5c7.4 0 10.8-9.2 5.2-14L286.9 550H872c4.4 0 8-3.6 8-8v-60c0-4.4-3.6-8-8-8z"
                />
              </svg>
            </span>
          </div>
        </div>
        <span
          class="ant-page-header-heading-title"
          title="Title"
        >
          Title
        </span>
        <span
          class="ant-page-header-heading-sub-title"
          title="This is a subtitle"
        >
          This is a subtitle
        </span>
        <span
          class="ant-page-header-heading-tags"
        >
          <span
            class="ant-tag ant-tag-blue"
          >
            Running
          </span>
        </span>
      </div>
      <span
        class="ant-page-header-heading-extra"
      >
        <button
          class="ant-btn"
          type="button"
        >
          <span>
            Operation
          </span>
        </button>
        <button
          class="ant-btn"
          type="button"
        >
          <span>
            Operation
          </span>
        </button>
        <button
          class="ant-btn ant-btn-primary"
          type="button"
        >
          <span>
            Primary
          </span>
        </button>
      </span>
    </div>
    <div
      class="ant-page-header-content"
    >
      <div
        class="ant-row"
      >
        <div
          class="ant-statistic"
        >
          <div
            class="ant-statistic-title"
          >
            Status
          </div>
          <div
            class="ant-statistic-content"
          >
            <span
              class="ant-statistic-content-value"
            >
              Pending
            </span>
          </div>
        </div>
        <div
          class="ant-statistic"
          style="margin:0 32px"
        >
          <div
            class="ant-statistic-title"
          >
            Price
          </div>
          <div
            class="ant-statistic-content"
          >
            <span
              class="ant-statistic-content-prefix"
            >
              $
            </span>
            <span
              class="ant-statistic-content-value"
            >
              <span
                class="ant-statistic-content-value-int"
              >
                568
              </span>
              <span
                class="ant-statistic-content-value-decimal"
              >
                .08
              </span>
            </span>
          </div>
        </div>
        <div
          class="ant-statistic"
        >
          <div
            class="ant-statistic-title"
          >
            Balance
          </div>
          <div
            class="ant-statistic-content"
          >
            <span
              class="ant-statistic-content-prefix"
            >
              $
            </span>
            <span
              class="ant-statistic-content-value"
            >
              <span
                class="ant-statistic-content-value-int"
              >
                3,345
              </span>
              <span
                class="ant-statistic-content-value-decimal"
              >
                .08
              </span>
            </span>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`renders ./components/page-header/demo/basic.md correctly 1`] = `
<div
  class="ant-page-header site-page-header ant-page-header-ghost"
>
  <div
    class="ant-page-header-heading"
  >
    <div
      class="ant-page-header-heading-left"
    >
      <div
        class="ant-page-header-back"
      >
        <div
          aria-label="Back"
          class="ant-page-header-back-button"
          role="button"
          style="border:0;background:transparent;padding:0;line-height:inherit;display:inline-block"
          tabindex="0"
        >
          <span
            aria-label="arrow-left"
            class="anticon anticon-arrow-left"
            role="img"
          >
            <svg
              aria-hidden="true"
              class=""
              data-icon="arrow-left"
              fill="currentColor"
              focusable="false"
              height="1em"
              viewBox="64 64 896 896"
              width="1em"
            >
              <path
                d="M872 474H286.9l350.2-304c5.6-4.9 2.2-14-5.2-14h-88.5c-3.9 0-7.6 1.4-10.5 3.9L155 487.8a31.96 31.96 0 000 48.3L535.1 866c1.5 1.3 3.3 2 5.2 2h91.5c7.4 0 10.8-9.2 5.2-14L286.9 550H872c4.4 0 8-3.6 8-8v-60c0-4.4-3.6-8-8-8z"
              />
            </svg>
          </span>
        </div>
      </div>
      <span
        class="ant-page-header-heading-title"
        title="Title"
      >
        Title
      </span>
      <span
        class="ant-page-header-heading-sub-title"
        title="This is a subtitle"
      >
        This is a subtitle
      </span>
    </div>
  </div>
</div>
`;

exports[`renders ./components/page-header/demo/breadcrumb.md correctly 1`] = `
<div
  class="ant-page-header site-page-header has-breadcrumb ant-page-header-ghost"
>
  <div
    class="ant-breadcrumb"
  >
    <span>
      <span
        class="ant-breadcrumb-link"
      >
        <a
          href="#/index"
        >
          First-level Menu
        </a>
      </span>
      <span
        class="ant-breadcrumb-separator"
      >
        /
      </span>
    </span>
    <span>
      <span
        class="ant-breadcrumb-link"
      >
        <a
          href="#/index/first"
        >
          Second-level Menu
        </a>
      </span>
      <span
        class="ant-breadcrumb-separator"
      >
        /
      </span>
    </span>
    <span>
      <span
        class="ant-breadcrumb-link"
      >
        <span>
          Third-level Menu
        </span>
      </span>
      <span
        class="ant-breadcrumb-separator"
      >
        /
      </span>
    </span>
  </div>
  <div
    class="ant-page-header-heading"
  >
    <div
      class="ant-page-header-heading-left"
    >
      <span
        class="ant-page-header-heading-title"
        title="Title"
      >
        Title
      </span>
      <span
        class="ant-page-header-heading-sub-title"
        title="This is a subtitle"
      >
        This is a subtitle
      </span>
    </div>
  </div>
</div>
`;

exports[`renders ./components/page-header/demo/content.md correctly 1`] = `
<div
  class="ant-page-header site-page-header has-breadcrumb ant-page-header-ghost"
>
  <div
    class="ant-breadcrumb"
  >
    <span>
      <span
        class="ant-breadcrumb-link"
      >
        <a
          href="#/index"
        >
          First-level Menu
        </a>
      </span>
      <span
        class="ant-breadcrumb-separator"
      >
        /
      </span>
    </span>
    <span>
      <span
        class="ant-breadcrumb-link"
      >
        <a
          href="#/index/first"
        >
          Second-level Menu
        </a>
      </span>
      <span
        class="ant-breadcrumb-separator"
      >
        /
      </span>
    </span>
    <span>
      <span
        class="ant-breadcrumb-link"
      >
        <span>
          Third-level Menu
        </span>
      </span>
      <span
        class="ant-breadcrumb-separator"
      >
        /
      </span>
    </span>
  </div>
  <div
    class="ant-page-header-heading"
  >
    <div
      class="ant-page-header-heading-left"
    >
      <span
        class="ant-avatar ant-avatar-circle ant-avatar-image"
      >
        <img
          src="https://avatars1.githubusercontent.com/u/8186664?s=460&v=4"
        />
      </span>
      <span
        class="ant-page-header-heading-title"
        title="Title"
      >
        Title
      </span>
      <span
        class="ant-page-header-heading-sub-title"
        title="This is a subtitle"
      >
        This is a subtitle
      </span>
      <span
        class="ant-page-header-heading-tags"
      >
        <span
          class="ant-tag ant-tag-blue"
        >
          Running
        </span>
      </span>
    </div>
    <span
      class="ant-page-header-heading-extra"
    >
      <button
        class="ant-btn"
        type="button"
      >
        <span>
          Operation
        </span>
      </button>
      <button
        class="ant-btn"
        type="button"
      >
        <span>
          Operation
        </span>
      </button>
      <button
        class="ant-btn ant-btn-primary"
        type="button"
      >
        <span>
          Primary
        </span>
      </button>
      <button
        class="ant-btn ant-dropdown-trigger"
        style="border:none;padding:0"
        type="button"
      >
        <span
          aria-label="ellipsis"
          class="anticon anticon-ellipsis"
          role="img"
          style="font-size:20px;vertical-align:top"
        >
          <svg
            aria-hidden="true"
            class=""
            data-icon="ellipsis"
            fill="currentColor"
            focusable="false"
            height="1em"
            viewBox="64 64 896 896"
            width="1em"
          >
            <path
              d="M176 511a56 56 0 10112 0 56 56 0 10-112 0zm280 0a56 56 0 10112 0 56 56 0 10-112 0zm280 0a56 56 0 10112 0 56 56 0 10-112 0z"
            />
          </svg>
        </span>
      </button>
    </span>
  </div>
  <div
    class="ant-page-header-content"
  >
    <div
      class="ant-row"
    >
      <div
        style="flex:1"
      >
        <div
          class="ant-typography"
        >
          Ant Design interprets the color system into two levels: a system-level color system and a product-level color system.
        </div>
        <div
          class="ant-typography"
        >
          Ant Design's design team preferred to design with the HSB color model, which makes it easier for designers to have a clear psychological expectation of color when adjusting colors, as well as facilitate communication in teams.
        </div>
        <div>
          <a
            class="example-link"
          >
            <img
              alt="Quick Start"
              class="example-link-icon"
              src="https://gw.alipayobjects.com/zos/rmsportal/MjEImQtenlyueSmVEfUD.svg"
            />
            Quick Start
          </a>
          <a
            class="example-link"
          >
            <img
              alt=" Product Info"
              class="example-link-icon"
              src="https://gw.alipayobjects.com/zos/rmsportal/NbuDUAuBlIApFuDvWiND.svg"
            />
             Product Info
          </a>
          <a
            class="example-link"
          >
            <img
              alt="Product Doc"
              class="example-link-icon"
              src="https://gw.alipayobjects.com/zos/rmsportal/ohOEPSYdDTNnyMbGuyLb.svg"
            />
            Product Doc
          </a>
        </div>
      </div>
      <div
        class="image"
      >
        <img
          alt="content"
          src="https://gw.alipayobjects.com/zos/antfincdn/K%24NnlsB%26hz/pageHeader.svg"
          width="100%"
        />
      </div>
    </div>
  </div>
</div>
`;

exports[`renders ./components/page-header/demo/ghost.md correctly 1`] = `
<div
  class="site-page-header-ghost-wrapper"
>
  <div
    class="ant-page-header"
  >
    <div
      class="ant-page-header-heading"
    >
      <div
        class="ant-page-header-heading-left"
      >
        <div
          class="ant-page-header-back"
        >
          <div
            aria-label="Back"
            class="ant-page-header-back-button"
            role="button"
            style="border:0;background:transparent;padding:0;line-height:inherit;display:inline-block"
            tabindex="0"
          >
            <span
              aria-label="arrow-left"
              class="anticon anticon-arrow-left"
              role="img"
            >
              <svg
                aria-hidden="true"
                class=""
                data-icon="arrow-left"
                fill="currentColor"
                focusable="false"
                height="1em"
                viewBox="64 64 896 896"
                width="1em"
              >
                <path
                  d="M872 474H286.9l350.2-304c5.6-4.9 2.2-14-5.2-14h-88.5c-3.9 0-7.6 1.4-10.5 3.9L155 487.8a31.96 31.96 0 000 48.3L535.1 866c1.5 1.3 3.3 2 5.2 2h91.5c7.4 0 10.8-9.2 5.2-14L286.9 550H872c4.4 0 8-3.6 8-8v-60c0-4.4-3.6-8-8-8z"
                />
              </svg>
            </span>
          </div>
        </div>
        <span
          class="ant-page-header-heading-title"
          title="Title"
        >
          Title
        </span>
        <span
          class="ant-page-header-heading-sub-title"
          title="This is a subtitle"
        >
          This is a subtitle
        </span>
      </div>
      <span
        class="ant-page-header-heading-extra"
      >
        <button
          class="ant-btn"
          type="button"
        >
          <span>
            Operation
          </span>
        </button>
        <button
          class="ant-btn"
          type="button"
        >
          <span>
            Operation
          </span>
        </button>
        <button
          class="ant-btn ant-btn-primary"
          type="button"
        >
          <span>
            Primary
          </span>
        </button>
      </span>
    </div>
    <div
      class="ant-page-header-content"
    >
      <div
        class="ant-descriptions ant-descriptions-small"
      >
        <div
          class="ant-descriptions-view"
        >
          <table>
            <tbody>
              <tr
                class="ant-descriptions-row"
              >
                <td
                  class="ant-descriptions-item"
                  colspan="1"
                >
                  <span
                    class="ant-descriptions-item-label ant-descriptions-item-colon"
                  >
                    Created
                  </span>
                  <span
                    class="ant-descriptions-item-content"
                  >
                    Lili Qu
                  </span>
                </td>
                <td
                  class="ant-descriptions-item"
                  colspan="1"
                >
                  <span
                    class="ant-descriptions-item-label ant-descriptions-item-colon"
                  >
                    Association
                  </span>
                  <span
                    class="ant-descriptions-item-content"
                  >
                    <a>
                      421421
                    </a>
                  </span>
                </td>
                <td
                  class="ant-descriptions-item"
                  colspan="1"
                >
                  <span
                    class="ant-descriptions-item-label ant-descriptions-item-colon"
                  >
                    Creation Time
                  </span>
                  <span
                    class="ant-descriptions-item-content"
                  >
                    2017-01-10
                  </span>
                </td>
              </tr>
              <tr
                class="ant-descriptions-row"
              >
                <td
                  class="ant-descriptions-item"
                  colspan="1"
                >
                  <span
                    class="ant-descriptions-item-label ant-descriptions-item-colon"
                  >
                    Effective Time
                  </span>
                  <span
                    class="ant-descriptions-item-content"
                  >
                    2017-10-10
                  </span>
                </td>
                <td
                  class="ant-descriptions-item"
                  colspan="2"
                >
                  <span
                    class="ant-descriptions-item-label ant-descriptions-item-colon"
                  >
                    Remarks
                  </span>
                  <span
                    class="ant-descriptions-item-content"
                  >
                    Gonghu Road, Xihu District, Hangzhou, Zhejiang, China
                  </span>
                </td>
              </tr>
            </tbody>
          </table>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`renders ./components/page-header/demo/responsive.md correctly 1`] = `
<div>
  <div
    class="ant-page-header site-page-header-responsive has-footer ant-page-header-ghost"
  >
    <div
      class="ant-page-header-heading"
    >
      <div
        class="ant-page-header-heading-left"
      >
        <div
          class="ant-page-header-back"
        >
          <div
            aria-label="Back"
            class="ant-page-header-back-button"
            role="button"
            style="border:0;background:transparent;padding:0;line-height:inherit;display:inline-block"
            tabindex="0"
          >
            <span
              aria-label="arrow-left"
              class="anticon anticon-arrow-left"
              role="img"
            >
              <svg
                aria-hidden="true"
                class=""
                data-icon="arrow-left"
                fill="currentColor"
                focusable="false"
                height="1em"
                viewBox="64 64 896 896"
                width="1em"
              >
                <path
                  d="M872 474H286.9l350.2-304c5.6-4.9 2.2-14-5.2-14h-88.5c-3.9 0-7.6 1.4-10.5 3.9L155 487.8a31.96 31.96 0 000 48.3L535.1 866c1.5 1.3 3.3 2 5.2 2h91.5c7.4 0 10.8-9.2 5.2-14L286.9 550H872c4.4 0 8-3.6 8-8v-60c0-4.4-3.6-8-8-8z"
                />
              </svg>
            </span>
          </div>
        </div>
        <span
          class="ant-page-header-heading-title"
          title="Title"
        >
          Title
        </span>
        <span
          class="ant-page-header-heading-sub-title"
          title="This is a subtitle"
        >
          This is a subtitle
        </span>
      </div>
      <span
        class="ant-page-header-heading-extra"
      >
        <button
          class="ant-btn"
          type="button"
        >
          <span>
            Operation
          </span>
        </button>
        <button
          class="ant-btn"
          type="button"
        >
          <span>
            Operation
          </span>
        </button>
        <button
          class="ant-btn ant-btn-primary"
          type="button"
        >
          <span>
            Primary
          </span>
        </button>
      </span>
    </div>
    <div
      class="ant-page-header-content"
    >
      <div
        class="content"
      >
        <div
          class="main"
        >
          <div
            class="ant-descriptions ant-descriptions-small"
          >
            <div
              class="ant-descriptions-view"
            >
              <table>
                <tbody>
                  <tr
                    class="ant-descriptions-row"
                  >
                    <td
                      class="ant-descriptions-item"
                      colspan="1"
                    >
                      <span
                        class="ant-descriptions-item-label ant-descriptions-item-colon"
                      >
                        Created
                      </span>
                      <span
                        class="ant-descriptions-item-content"
                      >
                        Lili Qu
                      </span>
                    </td>
                    <td
                      class="ant-descriptions-item"
                      colspan="1"
                    >
                      <span
                        class="ant-descriptions-item-label ant-descriptions-item-colon"
                      >
                        Association
                      </span>
                      <span
                        class="ant-descriptions-item-content"
                      >
                        <a>
                          421421
                        </a>
                      </span>
                    </td>
                  </tr>
                  <tr
                    class="ant-descriptions-row"
                  >
                    <td
                      class="ant-descriptions-item"
                      colspan="1"
                    >
                      <span
                        class="ant-descriptions-item-label ant-descriptions-item-colon"
                      >
                        Creation Time
                      </span>
                      <span
                        class="ant-descriptions-item-content"
                      >
                        2017-01-10
                      </span>
                    </td>
                    <td
                      class="ant-descriptions-item"
                      colspan="1"
                    >
                      <span
                        class="ant-descriptions-item-label ant-descriptions-item-colon"
                      >
                        Effective Time
                      </span>
                      <span
                        class="ant-descriptions-item-content"
                      >
                        2017-10-10
                      </span>
                    </td>
                  </tr>
                  <tr
                    class="ant-descriptions-row"
                  >
                    <td
                      class="ant-descriptions-item"
                      colspan="2"
                    >
                      <span
                        class="ant-descriptions-item-label ant-descriptions-item-colon"
                      >
                        Remarks
                      </span>
                      <span
                        class="ant-descriptions-item-content"
                      >
                        Gonghu Road, Xihu District, Hangzhou, Zhejiang, China
                      </span>
                    </td>
                  </tr>
                </tbody>
              </table>
            </div>
          </div>
        </div>
        <div
          class="extra"
        >
          <div
            style="display:flex;width:max-content;justify-content:flex-end"
          >
            <div
              class="ant-statistic"
              style="margin-right:32px"
            >
              <div
                class="ant-statistic-title"
              >
                Status
              </div>
              <div
                class="ant-statistic-content"
              >
                <span
                  class="ant-statistic-content-value"
                >
                  Pending
                </span>
              </div>
            </div>
            <div
              class="ant-statistic"
            >
              <div
                class="ant-statistic-title"
              >
                Price
              </div>
              <div
                class="ant-statistic-content"
              >
                <span
                  class="ant-statistic-content-prefix"
                >
                  $
                </span>
                <span
                  class="ant-statistic-content-value"
                >
                  <span
                    class="ant-statistic-content-value-int"
                  >
                    568
                  </span>
                  <span
                    class="ant-statistic-content-value-decimal"
                  >
                    .08
                  </span>
                </span>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="ant-page-header-footer"
    >
      <div
        class="ant-tabs ant-tabs-top ant-tabs-line"
      >
        <div
          class="ant-tabs-bar ant-tabs-top-bar"
          role="tablist"
          tabindex="-1"
        >
          <div
            class="ant-tabs-nav-container"
          >
            <span
              class="ant-tabs-tab-prev ant-tabs-tab-btn-disabled"
              unselectable="unselectable"
            >
              <span
                class="ant-tabs-tab-prev-icon"
              >
                <span
                  aria-label="left"
                  class="anticon anticon-left ant-tabs-tab-prev-icon-target"
                  role="img"
                >
                  <svg
                    aria-hidden="true"
                    class=""
                    data-icon="left"
                    fill="currentColor"
                    focusable="false"
                    height="1em"
                    viewBox="64 64 896 896"
                    width="1em"
                  >
                    <path
                      d="M724 218.3V141c0-6.7-7.7-10.4-12.9-6.3L260.3 486.8a31.86 31.86 0 000 50.3l450.8 352.1c5.3 4.1 12.9.4 12.9-6.3v-77.3c0-4.9-2.3-9.6-6.1-12.6l-360-281 360-281.1c3.8-3 6.1-7.7 6.1-12.6z"
                    />
                  </svg>
                </span>
              </span>
            </span>
            <span
              class="ant-tabs-tab-next ant-tabs-tab-btn-disabled"
              unselectable="unselectable"
            >
              <span
                class="ant-tabs-tab-next-icon"
              >
                <span
                  aria-label="right"
                  class="anticon anticon-right ant-tabs-tab-next-icon-target"
                  role="img"
                >
                  <svg
                    aria-hidden="true"
                    class=""
                    data-icon="right"
                    fill="currentColor"
                    focusable="false"
                    height="1em"
                    viewBox="64 64 896 896"
                    width="1em"
                  >
                    <path
                      d="M765.7 486.8L314.9 134.7A7.97 7.97 0 00302 141v77.3c0 4.9 2.3 9.6 6.1 12.6l360 281.1-360 281.1c-3.9 3-6.1 7.7-6.1 12.6V883c0 6.7 7.7 10.4 12.9 6.3l450.8-352.1a31.96 31.96 0 000-50.4z"
                    />
                  </svg>
                </span>
              </span>
            </span>
            <div
              class="ant-tabs-nav-wrap"
            >
              <div
                class="ant-tabs-nav-scroll"
              >
                <div
                  class="ant-tabs-nav ant-tabs-nav-animated"
                >
                  <div>
                    <div
                      aria-controls="tabpane-1"
                      aria-disabled="false"
                      aria-selected="true"
                      class="ant-tabs-tab-active ant-tabs-tab"
                      id="tab-1"
                      role="tab"
                      tabindex="0"
                    >
                      Details
                    </div>
                    <div
                      aria-controls="tabpane-2"
                      aria-disabled="false"
                      aria-selected="false"
                      class=" ant-tabs-tab"
                      id="tab-2"
                      role="tab"
                      tabindex="-1"
                    >
                      Rule
                    </div>
                  </div>
                  <div
                    class="ant-tabs-ink-bar ant-tabs-ink-bar-animated"
                  />
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          class="ant-tabs-content ant-tabs-content-animated ant-tabs-top-content"
          style="margin-left:0%"
        >
          <div
            aria-hidden="false"
            class="ant-tabs-tabpane ant-tabs-tabpane-active"
            role="tabpanel"
            style="visibility:visible"
            tabindex="0"
          />
          <div
            aria-hidden="true"
            class="ant-tabs-tabpane ant-tabs-tabpane-inactive"
            role="tabpanel"
            style="visibility:hidden"
            tabindex="-1"
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
